test/helper.lua

```diff
@@ -237,4 +237,21 @@ function helpers.is_expired_debug(_, tuple)
     return true
 end
 
+function helpers.tarantool_version()
+    local major_minor_patch = _G._TARANTOOL:split('-', 1)[1]
+    local major_minor_patch_parts = major_minor_patch:split('.', 2)
+
+    local major = tonumber(major_minor_patch_parts[1])
+    local minor = tonumber(major_minor_patch_parts[2])
+    local patch = tonumber(major_minor_patch_parts[3])
+
+    return major, minor, patch
+end
+
+function helpers.vinyl_is_broken()
+    -- Blocked by https://github.com/tarantool/tarantool/issues/6609
```

Comment:
AFAIK https://github.com/tarantool/tarantool/issues/6609 is a duplicate of https://github.com/tarantool/tarantool/issues/6448.

So, it's better to refer original issue.

Reply:
Okey, fixed.